firefox_puppeteer/ui/toolbars.py

```diff
@@ -6,13 +6,19 @@
 
 from ..api.keys import Keys
 from ..api.l10n import L10n
-from ..base import BaseLib
+from ..base import UIBaseLib
 from ..decorators import use_class_as_property
 
 
-class NavBar(BaseLib):
+class NavBar(UIBaseLib):
     """Provides access to the DOM elements contained in the
     navigation bar as well as the location bar."""
+    def __init__(self, marionette_getter, window):
+        UIBaseLib.__init__(self, marionette_getter, window)
+        
+        from .toolbars import LocationBar
```

Comment:
Why did you move this up here from the locationbar property? Please move it back.